README.md

```diff
@@ -1,4 +1,4 @@
-# An OOP school library app: Decorator pattern in OOP 🎁
+# An OOP school library app: Set up associations 🏹
 
```

Comment:
[OPTIONAL] Make sure you correctly set the MIT license to your project. When I tried to open it in the readme it shows me a broken link.